forest-node/README.md

```diff
@@ -0,0 +1,46 @@
+
+## Running Forest In DigitalOcean
+
+## Forest IAC Architecture
+
+![Forest Cloud Infrastructure ](https://user-images.githubusercontent.com/47984109/219903795-77c306b8-a70b-4f32-8d7d-3c1a39e52186.jpg)
+
+## Requirements  
+The droplet requirements to run forest-calibnet include:
+- RAM: 8GB
+- VCPU: 1
+- Disk Size: 100 GB
+The user local machine requirments include:
+- Install [terraform](https://developer.hashicorp.com/terraform/downloads) and [ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html).  
+- Install `make`
+- Basic DigitalOcean knowledge
+
+To implement the infrastructure, run the following:
+- Create `ssh-key` to be added to DigitalOcean list and store the fingerprint for use in the next step; you can check more details [here](https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/to-team/)
+- Create a space on DigitalOcean with any preferred unique name and add the bucket name in the `backend.tf` file. 
+- Generate `digitalocean_api_token` from DigitalOcean console; you can check [here](https://docs.digitalocean.com/reference/api/create-personal-access-token/) for more details.
+- Populate the `terraform.tfvars` file with the values of the following
+    - `new_key_ssh_key_fingerprint`
+    - `firewall_name`
+    - `name`
+- Set all necessary environment variables to the terminal permanently by adding the to a shell profile. To do this, run `nano ~/.bash_profile` then add the following to the end of the file
```

Comment:
I think we can just leave this out. Saying that the three environment variables need to be defined is good enough and then we don't have to mention `nano` or `.bash_profile`.

Reply:
Noted